schema/unsupported_features.json

```diff
@@ -2,27 +2,27 @@
     "setup_types": [
         {
             "name":"virtual",
-            "unsupported_features": ["lyve_pilot", "health"]
+            "unsupported_features": ["lyve_pilot", "health", "about"]
         },
         {
             "name":"VM",
-            "unsupported_features": ["lyve_pilot", "health"]
+            "unsupported_features": ["lyve_pilot", "health", "about"]
         },
         {
             "name":"other",
-            "unsupported_features": ["lyve_pilot", "health"]
+            "unsupported_features": ["lyve_pilot", "health", "about"]
         },
         {
             "name":"5u84",
-            "unsupported_features": ["lyve_pilot", "health"]
+            "unsupported_features": ["lyve_pilot", "health", "about"]
```

Comment:
Just some of things to clear @pawankumarsrivastava  @kupranay  @soniyamoholkar :
1. Are we blocking "about" section from HW as well??, because in HW, boxing-unboxing happens, so serial no. related bug wont occur.
2. If "about" section is hidden from UI then user wont ableee to see SSL details as well. But we are still able to see it from CLI (confirmed from Rohit).. 
So now from user's perpective SSL details are hidden in UI and visible in CLI.. Is it okay?

Reply:
@udayan-y21 
As per discussion with @kupranay, We are not blocking entire "about" section instead we have created new feature id - "appliance_info" for serial number api endpoint (/api/*/appliance_info) and we are blocking that from HW too.